pollbot/telegram/callback_handler/creation.py

```diff
@@ -53,20 +56,37 @@ def open_init_anonymization_settings(session, context, poll):
     open_anonymization_settings(context.query.message, poll)
 
 
+@poll_required
+def ask_description(session, context, poll):
+    """Asks user for description of the poll, in case the name was already supplied."""
+    context.user.expected_input = ExpectedInput.description.name
+    keyboard = get_skip_description_keyboard(poll)
+    context.tg_chat.send_message(
+        i18n.t("creation.description", locale=context.user.locale),
+        reply_markup=keyboard,
+    )
+
+
 @poll_required
 def skip_description(session, context, poll):
     """Skip description creation step."""
-    context.user.expected_input = ExpectedInput.options.name
-    session.commit()
-    context.query.message.edit_text(
-        i18n.t("creation.option.first", locale=context.user.locale),
-        reply_markup=get_open_datepicker_keyboard(poll),
-    )
+    if len(poll.options) == 0:
+        context.user.expected_input = ExpectedInput.options.name
+        session.commit()
+        context.query.message.edit_text(
+            i18n.t("creation.option.first", locale=context.user.locale),
+            reply_markup=get_open_datepicker_keyboard(poll),
+        )
+    else:  # options were already prefilled e.g. by native poll
+        # TODO: what if user changed poll type?
```

Comment:
After looking around a little bit, I don't see any problems with this.
Options don't have any special state depending on the poll type.

This should be ok.

Reply:
It's actually an artifact of my first attempt where I tried learning how to create a db migration, so now it's as simple as checking `poll.created_from_native` 🙈